src/commands/format.rs

```diff
@@ -98,20 +107,22 @@ fn read_source(path: &Path) -> Result<String> {
 
 /// Formats a document.
 ///
+/// Checks if the document is formatted correctly and prints the diff if not
+/// if check_only is true.
```

Comment:
```suggestion
/// If `check_only` is true, checks if the document is formatted correctly and prints the diff if not
/// then exits. Else will format and overwrite the document.
```
Just rewrote this in Github. Will require a run of `cargo +nightly fmt` after committing.